lv2/ttl/dual_flange.ttl

```diff
@@ -259,4 +260,4 @@
 <https://github.com/Stazed/rakarrack-plus#Dual_Flange:preset:preset2>
         a pset:Preset ;
         lv2:appliesTo <https://github.com/Stazed/rakarrack-plus#Dual_Flange> ;
-        rdfs:seeAlso <dual_flange_presets.ttl> .
```

Comment:
These are at a few places in lv2./ttl/ where diff said "No newline at end of file". I don't remember adding the missing newlines, but seems like diff output them anyway.